src/mon/MgrMonitor.cc

```diff
@@ -1102,10 +1135,9 @@ bool MgrMonitor::prepare_command(MonOpRequestRef op)
   cmd_getval(cmdmap, "format", format, string("plain"));
   boost::scoped_ptr<Formatter> f(Formatter::create(format));
 
-  string prefix;
-  cmd_getval(cmdmap, "prefix", prefix);
-
+  const auto prefix = cmd_getval_or<string>(cmdmap, "prefix", string{});
```

Comment:
Approved too soon -- this doesn't build:
```
/home/jenkins-build/build/workspace/ceph-api/src/mon/MgrMonitor.cc: In member function 'bool MgrMonitor::prepare_command(MonOpRequestRef)':
/home/jenkins-build/build/workspace/ceph-api/src/mon/MgrMonitor.cc:1138:23: error: 'cmd_getval_or' was not declared in this scope; did you mean 'cmd_getval'?
 1138 |   const auto prefix = cmd_getval_or<string>(cmdmap, "prefix", string{});
      |                       ^~~~~~~~~~~~~
      |                       cmd_getval
/home/jenkins-build/build/workspace/ceph-api/src/mon/MgrMonitor.cc:1138:43: error: expected primary-expression before '>' token
 1138 |   const auto prefix = cmd_getval_or<string>(cmdmap, "prefix", string{});
      |                                           ^
/home/jenkins-build/build/workspace/ceph-api/src/mon/MgrMonitor.cc:1138:45: warning: left operand of comma operator has no effect [-Wunused-value]
 1138 |   const auto prefix = cmd_getval_or<string>(cmdmap, "prefix", string{});
      |                                             ^~~~~~
/home/jenkins-build/build/workspace/ceph-api/src/mon/MgrMonitor.cc:1138:70: warning: right operand of comma operator has no effect [-Wunused-value]
 1138 |   const auto prefix = cmd_getval_or<string>(cmdmap, "prefix", string{});
      |                                                                      ^
make[2]: *** [src/mon/CMakeFiles/mon.dir/build.make:202: src/mon/CMakeFiles/mon.dir/MgrMonitor.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
```

Reply:
Should be fixed now.